content/sips/sip-192.md

```diff
@@ -7,7 +7,7 @@ author: Daniel Beal (@dbeal-eth)
 implementor: Daniel Beal (@dbeal-eth)
 release: 12-2-2021
 discussions-to: 'https://research.synthetix.io'
-created: 2021-11-18T00:00:00.000Z
+created: 2021-12-02T00:00:00.000Z
```

Comment:
probs put the a new field here @drptbl 

created: 2021-11-18T00:00:00.000Z
updated: 2021-12-02T00:00:00.000Z

Reply:
@andytcf are you sure?
It wasn't really created on 2021-11-18 afaik.